app/javascript/app/document-capture/components/acuant-capture.jsx

```diff
@@ -9,7 +8,23 @@ import useI18n from '../hooks/use-i18n';
 import DeviceContext from '../context/device';
 import DataURLFile from '../models/data-url-file';
 
-function AcuantCapture({ label, bannerText, value, onChange, className }) {
+/**
+ * @typedef AcuantCaptureProps
+ *
+ * @prop {string}                        label      Label associated with file input.
+ * @prop {string=}                       bannerText Optional banner text to show in file input.
+ * @prop {DataURLFile=}                  value      Current value.
+ * @prop {(nextValue:DataURLFile)=>void} onChange   Callback receiving next value on change.
+ * @prop {string=}                       className  Optional additional class names.
+ */
+
+/**
+ * Returns an element serving as an enhanced FileInput, supporting direct capture using Acuant SDK
+ * in supported devices.
+ *
+ * @param {AcuantCaptureProps} props Props object.
```

Comment:
I'd hoped instead to do....

```suggestion
 * @type {import('react).FC<AcuantCaptureProps>}
```

...which, from what I can tell, should be a valid way to annotate this as a function component with the set of props, and has an advantage of auto-handling some details (like return value type, merging `children` into `props`, etc). Maybe it's a shortcoming of the inline `// @ts-check` that it wasn't working. Might be worth revisiting if/when type-checking is enabled project-wide.